CONTRIBUTING.md

```diff
@@ -111,4 +111,53 @@ When you have successfully run the command to put this local version on your mac
 
-When completed, you can redeploy the 2 Liferay modules, `frontend-js` and `frontend-theme` to test your changes.
\ No newline at end of file
+When completed, you can redeploy the 2 Liferay modules, `frontend-js` and `frontend-theme` to test your changes.
+
+## Release
+
+Alloy UI is used in Liferay Portal via [Liferay nexus repository](https://repository.liferay.com/nexus/content/repositories/liferay-public-releases). 
+
+### Maven permissions
+
+To publish a new version in nexus we need permissions to create files under that repository. We can ask for permission to create a ticket in JIRA.
+
+Once we have permissions we have to configure our maven installation to use those credentials by modyfying the settings.xml file that is usually located under `~USER_FOLDER/.m2/settings.xml` and add a new server block with these values:
+
+```
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <servers>
+    <server>
+      <id>liferay-public-snapshots</id>
+      <username>YOUR_USERNAME</username>
+      <password>YOUR_CREDENTIALS</password>
+    </server>
+    <server>
+      <id>liferay-public-releases</id>
+      <username>YOUR_USERNAME</username>
+      <password>YOUR_CREDENTIALS</password>
+    </server>
+  </servers>
+</settings>
+```
+
+### Steps to publish a test version
+
+If everything is set up properly we can create a snapshot by executing:
+
+* `npm install` (if you have not done it already)
+* `gulp build` (if you have not done it already)
+* `gulp maven-publish-snapshot` 
+
+This should create a jar under this [folder](https://repository.liferay.com/nexus/content/repositories/liferay-public-snapshots/com/liferay/webjars/com.liferay.webjars.alloy-ui/). The size of the binary should be around 25MB. You can publish as many times as you want under that folder (and it will rename the binary to have a numeric suffix like -2.jar).
+
+### Common issues
+
+If the size is way smaller than 25MB it could mean that you have missed a step (most probably `gulp build`). If the size is way bigger than 25MB it is probably because the publish script is picking local files under the `alloy-ui` folder. Remove binary files/jar generated by `maven install` or move the `.git` folder outside the main folder while you are generating. 
```

Comment:
Moving `.git` folder? :scream: If the script that builds `jar` grabs the `.git` folder, that sounds like a bug to me. 

`maven install` I can understand, if we expect `maven` to be installed globally. If that's the case, we should probably commit it to the repo. Or, at the very least, document global install as one of the release steps.